lib/KTable/index.vue

```diff
@@ -342,16 +342,16 @@
                   nextColIndex = totalCols - 1;
                   nextRowIndex = rowIndex - 1;
                 } else {
-                  nextColIndex = totalCols - 1;
-                  nextRowIndex = totalRows - 1;
+                  // Allow default behavior when reaching the first cell
+                  return;
                 }
               }
             }
-            this.focusCell(nextRowIndex, nextColIndex);
+            this.focusCell(nextRowIndex, nextColIndex, true);
```

Comment:
What was the intention behind the third `true` parameter here? I don't see it being used in the function.

Reply:
I was trying a different approach to implement the tab key behaviour earlier that involved use of this 'true' prop. Seems I forgot to clean it up :